pkg/controllers/mpi_job_controller.go

```diff
@@ -485,7 +487,7 @@ func allocateGPUs(mpiJob *kubeflow.MPIJob, gpusPerNode int, done bool) (workerRe
 		if totalGPUs < gpusPerNode {
 			workerReplicas = 1
 			gpusPerWorker = totalGPUs
-		} else if totalGPUs % gpusPerNode == 0 {
+		} else if totalGPUs%gpusPerNode == 0 {
```

Comment:
Personally, I prefer the previous way.

Reply:
It's changed by `go fmt` not me. :)

Please check https://goreportcard.com/report/github.com/kubeflow/mpi-operator

Reply:
Oh my fault, LGTM